### ISSUE TITLE
[🐞]: Failed to decode packet 'clientbound/minecraft:map_item_data'

### ISSUE THREAD

**Issue by onycius** [MehVahdJukaar/Moonlight#253]
### BEFORE CONTINUING:

- [X] Always check you are using the latest version of these mods and its dependencies
- [X] Remove known problematic rendering mods such as Optifine, Sodium and others

### Version-Loader

1.21-fabric

### Moonlight Lib Version

moonlight-1.21-2.14.17

### Issue Detail

I am immediately kicked when attempting to join minecraft servers with the latest moonlight lib installed on the client. No other mods are installed.

### Log Attachment

[disconnect-2024-09-10_23.55.46-client.txt](https://github.com/user-attachments/files/16959187/disconnect-2024-09-10_23.55.46-client.txt)

[latest.log](https://github.com/user-attachments/files/16959191/latest.log)

### OPTIONAL: To Produce

1) Install moonlight lib
2) join a minecraft server
3) get kicked

**Comment by MehVahdJukaar** [comment]
does the server have the mod?

**Comment by onycius** [comment]
yes the server has the mod installed. I also attempted joining a non-modded server like Hypixel and the kick still occurs.

**Comment by MehVahdJukaar** [comment]
well non modded server is expected. Can you verity the version on server and client match?

**Comment by MehVahdJukaar** [comment]
also is the server a normal forge server? I just tried on a server with just the mod, gave myself a map and no crash occurred

**Comment by Noctree** [comment]
Most likely same issue, Minecraft 1.21.1, NeoForge: 21.1.148

Server running also NeoForge 21.1.148 with moonlight-1.21-2.18.13-neoforge installed, same version as client

Disconnect log:

[disconnect-2025-05-04_23.17.57-client.txt](https://github.com/user-attachments/files/20030062/disconnect-2025-05-04_23.17.57-client.txt)

**Comment by MehVahdJukaar** [comment]
You will need to provide more info. As I've said if it doesn't happen with just my kids it's a clash with another mod
